### ISSUE TITLE
Are we moving too fast for auto-exposure to keep up?

### ISSUE THREAD

**Issue by chrissearle** [dbaldwin/DronePan#37]
Need to check - do we need just a little longer after a move before taking a photo for those using auto-exposure mode? Some reports of over-exposure after moving from down to point skywards.

**Comment by prestondillon86** [comment]
nah, from what I understand there are almost zero circumstances that they should be shooting in autoexposure. At one point I was even suggesting that we force locked exposure, but Dennis made a good point that then people might forget to check other settings and end up with an entire set of images that is blown out or under exposed.
